example/Example.hs

```diff
@@ -41,7 +41,7 @@ main =
          do makeContextCurrent win
             glewInit
             glGetError
-            c@(Context c') <- createGL3 (S.fromList [Antialias,StencilStrokes,Debug])
+            c@(Context c') <- createGLES3 (S.fromList [Antialias,StencilStrokes,Debug])
```

Comment:
Why change the example? It seems like GL3 as the default is quite reasonable.

Reply:
Oh. Sorry. I didn't mean to commit that change...